src/main/java/org/spongepowered/common/data/builder/manipulator/mutable/entity/ElderDataBuilder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.builder.manipulator.mutable.entity;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.data.DataHolder;
+import org.spongepowered.api.data.DataView;
+import org.spongepowered.api.data.key.Keys;
+import org.spongepowered.api.data.manipulator.DataManipulatorBuilder;
+import org.spongepowered.api.data.manipulator.immutable.entity.ImmutableElderData;
+import org.spongepowered.api.data.manipulator.mutable.entity.ElderData;
+import org.spongepowered.api.service.persistence.InvalidDataException;
+import org.spongepowered.common.data.manipulator.mutable.entity.SpongeElderData;
+
+public class ElderDataBuilder implements DataManipulatorBuilder<ElderData, ImmutableElderData> {
+
+    @Override
+    public Optional<ElderData> build(DataView container) throws InvalidDataException {
+        if (container.contains(Keys.ELDER_GUARDIAN.getQuery())) {
+            Optional<Boolean> elder = container.getBoolean(Keys.ELDER_GUARDIAN.getQuery());
+            if (elder.isPresent()) {
```

Comment:
Yes. Any PR that isn't taking advantage of the various reductions in boilerplate code with `DataProcessor`s in general I'm requiring to be updated. I've considered nagging, if not just handling the issues myself, but I want the contributors to understand exactly what they need to be doing to implement the processors correctly.